src/c++/library/CMakeLists.txt

```diff
@@ -327,7 +327,7 @@ if(TRITON_ENABLE_CC_HTTP OR TRITON_ENABLE_PERF_ANALYZER)
   target_link_libraries(
       httpclient
       PRIVATE triton-common-json
-      PRIVATE CURL::libcurl
+      PUBLIC CURL::libcurl
```

Comment:
`cc_client_test` cmake library includes `http_client.cc`, which includes `<curl/curl.h>` and calls symbols from it. This allows the curl symbols to be called for the `httpclient_static` cmake library and all targets that use `httpclient_static` (`cc_client_test`).